apps/site/next-data/generators/supportersData.mjs

```diff
@@ -0,0 +1,95 @@
+// This is used to ensure that URLs are always in the correct format
+function fixUrl(url) {
+  if (!url) {
+    return null;
+  }
+
+  if (!url.startsWith('http://') && !url.startsWith('https://')) {
+    return `https://${url}`;
+  }
+
+  return url;
+}
+
+async function fetchOpenCollectiveData() {
+  const endpoint = 'https://api.opencollective.com/graphql/v2';
+
+  const query = `{
```

Comment:
Could you use their REST API please? https://docs.opencollective.com/help/contributing/development/api I'd like to remove the need of adding yet another layer of complexity (GraphQL) here